index.template.html

```diff
@@ -40,6 +40,7 @@
 <link rel="manifest" href="/manifest.json" />
 <meta name="google-signin-client_id" content="912270888098-jjnre816lsuhc5clj3vbcn4o2q7p4qvk.apps.googleusercontent.com">
 <script>var Config = {};</script>
+<script src="//play.pokemonshowdown.com/config/config.js?"></script>
```

Comment:
Maybe not _this_ far up. Under `"Loading libraries..." but before the first library looks like the correct place.

Also is `var Config = {}` still necessary? I think it was previously to guard against Config not existing, but now that we have a hard dependency on `Config.routes`, I don't think it's necessary anymore.

Reply:
Hardcoding an empty `Config` seems silly when we include `config.js` *and* proceed to depend upon things that aren't hardcoded in it. If it were necessary, I'd want to hardcode default routes there as well (though I'm fine with just removing it).

Reply:
I agree. Let's remove the hardcode.